docs/source/data-model.rst

```diff
@@ -97,7 +165,46 @@ Typical example:
     'time': 1550070004.9850419,
     'uid': 'ba1f9076-7925-4af8-916e-0e1eaa1b3c47'}
 
-Formal schema:
+*end remove*
+
+The command:
+
+.. code-block:: python
+
+    from bluesky.plans import scan
+    from ophyd.sim import det, motor  # simulated detector, motor
+
+    # Scan 'motor' from -3 to 3 in 10 steps, taking readings from 'det'.
+    RE(scan([det], motor, -3, 3, 16), purpose='calibration',
+       sample='kryptonite')
+
+generates a typical 'run start' document like this:
+
+.. code-block:: python
```

Comment:
This is older and less accurate that the version in event-model. We should keep the version in event-model. We could however reincorporate the example bluesky code if that seems helpful.

Reply:
check